index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/style.css" type="text/css" rel="stylesheet">
+    <link href="./assets/font.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+    <div class="header">
+        <img src="assets/logo.png" class="logo" id="logo">
+        <ul class="horizontal-list">
+            <li><a href="#howitworks"><span id="word-spacing">How it works</span></a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+        </ul>
+    </div>
+    <div class="one">
+        <div class="head">
+            <div class="headline">
+                <h1 id="headlineSAFOR">SAFOR</h1>
+                <h1>switching your<br>
+                    life to safety mode</h1>
+                <p>Safor is an intelligent personal guard platform that empowers individuals to enjoy active and safe
+                    lives,<br>
+                    while their friends and family can enjoy peace of mind.</p>
+            </div>
+            <div id="hrs">
+                <hr id="hr1">
+                <hr id="hr2">
+                <hr id="hr3">
+                <hr id="hr4">
+                <hr id="hr5">
+                <hr id="hr6">
+                <hr id="hr7">
+            </div>
+        </div>
+    </div>
+    <div class="two">
+        <div id="about">
+            <div>
+                <h1>WHAT IS SAFOR?</h1>
+                <div>
+                    <img src="assets/Rectangle.svg">
+                    <h3 id="abouth3"><a href=" ">WATCH NOW</a></h3>
```

Comment:
Please use underscore or dash to separate two different tags. Same for all Id's and Classes in document.